powershell/VstsTaskSdk/ToolFunctions.ps1

```diff
@@ -117,8 +117,8 @@ function Invoke-Tool { # TODO: RENAME TO INVOKE-PROCESS?
         }
 
         $FileName = $FileName.Replace('"', '').Replace("'", "''")
-        $expression = "& '$FileName' --% $Arguments"
-        Invoke-Expression $expression
+        Write-Host """$FileName"" $Arguments"
+        Invoke-Expression "& '$FileName' --% $Arguments"
```

Comment:
This change prints the command line before invoking it. I deleted the intermediate variable since it wasn't needed. The printed command line is not exactly the same as the PowerShell expression that is invoked. The fact that PowerShell is used to launch the command line is an internal detail, the user should see they command line they expect. The `--%` operator is not widely known, so it would just be confusing to show the end user that syntax.